reports/aksversions/README.md

```diff
@@ -8,50 +8,68 @@ The scripts goal is to find every AKS cluster within a set of subscriptions, fin
 
 There are 2 scripts that make this report work:
 
-- `aks-versions.sh` - This script does the search and json build
+- `main.py` - This script searches Azure for AKS clusters and creates a JSON object of relevant information.
     - Searches for all subscriptions containining `SHAREDSERVICES` or `CFT` and addes the subscription Id into an array
     - Uses the list of subscriptions to then search for deployed AKS clusters and adds this information to another array
     - Checks the discovered AKS clusters for available updates
     - Builds an new json object containing all the relevant information so it can be added to a Cosmos container
-- `save-to-cosmos.py` - This script is only used to interact with Cosmos DB
-    - Will remove all documents from the chosen container
-    - Will then add all new documents supplied to the script from the `aks-versions.sh` script
+- `cosmos_functions.py.py` - This script is only used to interact with Cosmos DB and provides functions to do so that are imported to main.py
```

Comment:
```suggestion
- `cosmos_functions.py` - This script is only used to interact with Cosmos DB and provides functions to do so that are imported to main.py
```